src/ansys/units/variable_descriptor/variable_descriptor.py

```diff
@@ -103,13 +103,13 @@ class VariableCatalog:
     @classmethod
     def all(cls) -> dict[str, list[VariableDescriptor]]:
         """
-        Return all defined `VariableDescriptor`s, organized by subcategory.
+        Return all defined VariableDescriptors, organized by subcategory.
```

Comment:
Something here was triggering warning during doc build. The warning went away after the change (but build warnings randomly disappear and re-appear all the time!).